SignalMessaging/categories/UIColor+OWS.m

```diff
@@ -304,6 +270,50 @@ + (UIColor *)ows_grey600Color
     return [UIColor colorWithRGBHex:0x757575];
 }
 
++ (NSDictionary<NSString *, UIColor *> *)ows_conversationColorMap
+{
+    static NSDictionary<NSString *, UIColor *> *colorMap;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        colorMap = @{
+                     @"red" : self.ows_red700Color,
+                     @"pink": self.ows_pink600Color,
+                     @"purple": self.ows_purple600Color,
+                     @"indigo": self.ows_indigo600Color,
+                     @"blue": self.ows_blue700Color,
+                     @"cyan": self.ows_cyan800Color,
+                     @"teal": self.ows_teal700Color,
+                     @"green": self.ows_green800Color,
+                     @"deep_orange": self.ows_deepOrange900Color,
+                     @"grey": self.ows_grey600Color
+                     };
+    });
+    
+    return colorMap;
+}
+
++ (NSArray<NSString *> *)ows_conversationColorNames
+{
+    return self.ows_conversationColorMap.allKeys;
```

Comment:
If you take my suggestion above to key the color picker on "color names" instead of "color values", we could sort here to ensure that they are presented with a consistent ordering.